<!-- Launch file for the control software in the loop testing node -->


<launch>
  <!-- type-> executable , pkg->package, name-> node name-->
  <node type="control_sil_sim" pkg="control_sil" name="control_sil_sim" output="screen" >
    <!-- Please note that 0.001 is the optimal value for simulation time step, smaller time step may cause system instability and slowness -->
    <!-- Time is in seconds, wheelbase and positions are in meters, heading is in radians-->
    <param name="simulation_time_step" value="0.001"/>
    <param name="vehicle_wheelbase" value="2.3622"/>
    <param name="initial_x_position" value="147"/>
    <param name="initial_y_position" value="247"/>
    <param name="initial_vehicle_heading" value="-1.0"/>
    <param name="use_sil_hardcoded_planner" value="false"/>
  </node>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find control_sil)/rviz/control_sil_rviz.rviz" />
</launch>
